### PR TITLE
fix [Accessibility] Accessibility Insights tool keeps loading when mouse-over DataGridView #10343

### LINKED ISSUE #10343: [Accessibility] Accessibility Insights tool keeps loading when mouse-over DataGridView
### .NET version

9.0.100-alpha.1.23564.26

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

It works in .Net 7.0, doesn't in .Net 8.0

### Issue description

If mouse-click textBox cell in DataGridView, open AccessibilityInsight, then mouse-over DataGridView cell. It leads to AccessibilityInsight keeps loading.

**.Net 9.0/.Net 8.0: AccessibilityInsight keeps loading**
DataGridView_AccessibilityInsightIssue

**.Net 7.0: AccessibilityInsight works well**
DataGridView_NET7 0

### Steps to reproduce

1. Create a Winforms .Net 9.0/8.0 application
2. Add a DataGridView with TextBox cell
3. Build and run application
4. Mouse-click in DataGridView_textBox cell
5. Launch AccessibilityInsight, then mouse-over DataGridView

### PR DESCRIPTION
**Pull request by Epica3055**
Fix #10343

Same change from [PR_10372](https://github.com/dotnet/winforms/pull/10372)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10472)

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
@Epica3055  - had this been tested by Olina's team?

File: src/System.Windows.Forms/src/PublicAPI.Unshipped.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*REMOVED*override System.Windows.Forms.DataGridViewTextBoxEditingControl.OnHandleCreated(System.EventArgs! e) -> void
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/PublicAPI.Unshipped.txt:1]: @lonitra @JeremyKuhne  - I want to double-check that removal of a public override is allowed in servicing.

Comment (JeremyKuhne) [reply src/System.Windows.Forms/src/PublicAPI.Unshipped.txt:1]: I think it's ok, but I'd just leave it in and call base so we don't touch this file.

Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/src/PublicAPI.Unshipped.txt:1]: @Epica3055  - please make the suggested change and send email to tactics.

**Comment by Epica3055** [comment]
> @Epica3055 - had this been tested by Olina's team?

@Tanya-Solyanik yes. It had.

https://teams.microsoft.com/l/message/19:df1b45ad91934a4d890f3d50f77c8693@thread.skype/1702436024904?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=fd4e6d72-40d6-4488-ad7c-ca5727c5c026&parentMessageId=1702436024904&teamName=WinForms%2FClickOnce&channelName=Test%20Requests&createdTime=1702436024904

**Comment by lonitra** [comment]
February servicing is not open yet, this will need to wait until then for merge

**Comment by Tanya-Solyanik** [comment]
We can still send an email to tactics and have the approval ready for the February merge. @Epica3055  - Please make the changes
```
 protected override void OnHandleCreated(EventArgs e)
    {
        base.OnHandleCreated(e);
    }
```
And send the e-mail

**Pull request merged**